sympy/sets/tests/test_sets.py

```diff
@@ -1065,3 +1065,13 @@ def test_issue_11174():
 
     soln = Intersection(S.Reals, FiniteSet(x), evaluate=False)
     assert Intersection(FiniteSet(x), S.Reals) == soln
+
+
+def test_issue_13956():
+    assert S.EmptySet.is_empty == True
+    assert Interval(0, 1).is_empty == False
+    assert Interval(0, 0).is_empty == None
+    assert Interval(0, 0, left_open=True, right_open=True).is_empty == True
+    assert Union(Interval(0, 1), Interval(0, 1)).is_empty == False
+    assert Union(Interval(1, 0), Interval(1, 0)).is_empty == True
```

Comment:
There should be tests with different kinds of arguments. Here both arguments are EmptySets and so the union is also EmptySet.
```
>>> Interval(1, 0)
EmptySet()
>>> Union(Interval(1, 0), Interval(1, 0))
EmptySet()
```
Therefore this does not test `Union.is_empty()` at all.

Reply:
But an `Union` can only be `empty` if we take the union of two `EmptySet()`.
In all other cases, it will possibly return `False` or `None`.
Any examples you have on mind? @jksuom

Reply:
The `Union` could be created with `evaluate=False`. Also different kinds of empty sets could appear in addition to empty intervals, `FiniteSet()`, for example.